pkg/internal/kubectl/kubectl_test.go

```diff
@@ -24,13 +24,13 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/fidelity/kraan/pkg/internal/kubectl"
-	mocks "github.com/fidelity/kraan/pkg/internal/mocks/kubectl"
-	mocklogr "github.com/fidelity/kraan/pkg/internal/mocks/logr"
-
 	"github.com/go-logr/logr"
 	testlogr "github.com/go-logr/logr/testing"
 	gomock "github.com/golang/mock/gomock"
+
+	"github.com/fidelity/kraan/pkg/internal/kubectl"
+	mocks "github.com/fidelity/kraan/pkg/internal/mocks/kubectl"
+	mocklogr "github.com/fidelity/kraan/pkg/internal/mocks/logr"
```

Comment:
Our respective formatters keep moving these blocks back and forth.  We need to reconcile that soon so we don't keep reverting this diff in each others' commits.

Reply:
I've updated my goimports version, lets sync on this